src/Xamarin.Android.Build.Tasks/Tasks/ConvertResourcesCases.cs

```diff
@@ -71,7 +78,25 @@ void FixupResources (ITaskItem item, Dictionary<string, string> acwMap)
 				MonoAndroidHelper.SetWriteable (tmpdest);
 				try {
 					AndroidResource.UpdateXmlResource (resdir, tmpdest, acwMap,
-						ResourceDirectories.Where (s => s != item).Select(s => s.ItemSpec));
+						ResourceDirectories.Where (s => s != item).Select(s => s.ItemSpec), (t, m) => {
+							string targetfile = file;
+							if (targetfile.StartsWith (resdir, StringComparison.InvariantCultureIgnoreCase)) {
+								targetfile = file.Substring (resdir.Length).TrimStart (Path.DirectorySeparatorChar);
+								targetfile = resource_name_case_map.ContainsKey (targetfile) ? resource_name_case_map [targetfile] : targetfile;
```

Comment:
not sure. We would need to add a temp variable since it seems `TryGetValue` will `null` the output if it failed.

`When this method returns, contains the value associated with the specified key, if the key is found; otherwise, the default value for the type of the value parameter. This parameter is passed uninitialized.`

 So we can't do 

```
resource_name_case_map.TryGetValue (targetfile, out targetfile);
```

we would need to do

```
string temp;
if (resource_name_case_map.TryGetValue (targetfile, out temp))
    targetfile = temp;
```

not sure which is better.

Reply:
ah, stuff it. Gonna change to `TryGetValue`

Reply:
I think this could make it a little better:
```
if (resource_name_case_map.TryGetValue (targetfile, out string temp))
    targetfile = temp;
```